src/test/acceptance/features/confirmation-code-navigation.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Confirmation code navigation
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  Once I have entered my confirmation code
+  If I change the contact details used to receive that code
+  I have to go through the send and enter code part of the application to receive a new code.
+
+  Background:
+    Given I am on the first page of the application
+
+  Scenario: Changing my phone number after using it to receive a code requires me to be sent a new code
+    When I complete the application with valid details, selecting to receive my confirmation code via text
+    And I choose to change my answer to phone number
```

Comment:
How about `I choose to change my phone number`?

Reply:
Reworded.